### PR TITLE
[8.0] Add website recruitment firstname

### PR DESCRIPTION
**Pull request by adrienpeiffer**
- Add firstname on the website recruitment form
- [ ] Depends on https://github.com/OCA/hr/pull/129
- [x] Depends on https://github.com/odoo/odoo/pull/8049

### CODE REVIEW

File: website_hr_recruitment_firstname/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    'summary': 'Adds First Name for website recruitment',
+    'depends': [
+        'hr_recruitment_firstname',
+        'website_hr_recruitment',
+    ],
+    'data': [
+        'views/hr_applicant_template.xml',
+    ],
+    'installable': True,
+    'auto_install': True,
```
Comment (andreparames) [inline website_hr_recruitment_firstname/__openerp__.py]: Why is this module set to auto-install?

File: website_hr_recruitment_firstname/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+
+class WebsiteHrRecruitment(website_hr_recruitment):
+
+    def _get_applicant_char_fields(self):
+        res = super(WebsiteHrRecruitment, self)._get_applicant_char_fields()
+        fields_list = set(['partner_firstname'])
+        res = set(res)
+        res |= fields_list
+        return list(res)
```
Comment (andreparames) [inline website_hr_recruitment_firstname/controllers/main.py]: Any reason not to do

    res = super(WebsiteHrRecruitment, self)._get_applicant_char_fields()
    res.remove('partner_firstname')
    return res

?

Comment (andreparames) [inline website_hr_recruitment_firstname/controllers/main.py:34]: Any reason not to do

```
if 'partner_firstname' not in res:
  res.append('partner_firstname')
return res
```

?

**Comment by rafaelbn** [comment]
Waiting for https://github.com/OCA/hr/pull/129 so WIP

**Comment by dreispt** [comment]
My impression is that this module is too trivial: it just adds a field to a form, and this particular form will often be customized as per specific customer needs.
I don't think this is worth a module.

**Comment by rafaelbn** [comment]
Let's see what @adrienpeiffer says. This is a 14th august 2015 module

**Comment by simahawk** [comment]
@adrienpeiffer are you still working on this?
I'm closing as your last action is from 2015 :)
Feel free to reopen it if needed, thanks!

**Pull request closed without merging**